config.ts

```diff
@@ -0,0 +1,12 @@
+import { randomBytes } from "crypto";
+
+const generateRandomKey = () => {
+    const randomBytesResult = randomBytes(32);
+    return randomBytesResult.toString("hex");
+};
+
+const config = {
+    APP_SECRET_KEY: generateRandomKey()
```

Comment:
You'll still need to check for your environment variable, if that doesn't exist then you need generate a random key.